### DISCUSSION TITLE
NameError: name 'tasks' is not defined

### DISCUSSION THREAD

**Discussion by moon09I980616I** [embeddings-benchmark/mteb#148] [Q&A] [answered]
from mteb import MTEB
from sentence_transformers import SentenceTransformer

# Define the sentence-transformers model name
model_name = "average_word_embeddings_komninos"

model = SentenceTransformer(model_name)
evaluation = MTEB(tasks=["Banking77Classification"])
results = evaluation.run(model, output_folder=f"results/{model_name}")

hello. I am running the code using the mteb dataset. However, today, suddenly, at this part of "evaluation = MTEB(tasks=["Banking77Classification"])", an error occurs: "NameError: name 'tasks' is not defined." Can you tell me why?
The code above is the code I wrote accordingly.

**Comment by Muennighoff** [comment]
Hmm, I cannot reproduce the error. Could you share a Colab Notebook where the error can be reproduced?

**Reply by moon09I980616I** [reply, marked as answer]
Hello, pip install mteb has different parts from mteb.py and GitHub, so I uninstalled it and entered the command "pip install git+https://github.com/embeddings-benchmark/mteb-draft.git" and the error disappeared. I lost.
